### DISCUSSION TITLE
A way to add "updates" (a log) for items

### DISCUSSION THREAD

**Discussion by Pikalugia** [benjaminjonard/koillection#917] [Ideas]
I am new to Koillection, great project by the way! I don't know if this is already a feature but it'd be great to have a log for each item where I can post updates for said item. Example: If I decided to sell my game I can put an update on the item that it's sold at this date and then bought back at another date - or something like that.

Quickly threw together a concept.

![updates](https://github.com/benjaminjonard/koillection/assets/75184384/de7ecd2f-aab3-4a84-b3de-b3a3df645b94)

**Comment by mikeymakesit** [comment]
I've just started using Koillection so I'm the furthest possible thing from an expert, however the way I'm addressing bought/sold info is to add purchase info (where/when bought and purchase price) to my collection item template, and I have another template for sales.  When I sell something I can edit it to add fields from the other template and fill them in.  Then I get the bought/sold date, from/to whom, and prices.

**Comment by azuravian** [comment]
It's not perfect, but you could add an entry to your template called Updates that has a Type of List.  Then just add an entry to the list each time you want to add an update entry.  Each update would just be a string, so it wouldn't automatically timestamp, but you could just enter the date at the beginning of your string, like your mockup.
